airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/spec.json

```diff
@@ -98,6 +98,16 @@
         "examples": ["2021-01-01"],
         "order": 2
       },
+      "date_ranges_end_date": {
```

Comment:
could you add test with `date_ranges_end_date` add to unit tests

Reply:
Added the tests, thanks for the hint